### DISCUSSION TITLE
EddieHub Shorts!

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#1148] [Ideas]
As discussed on today's [live stream](https://www.youtube.com/watch?v=djpH43hsOJI), @eddiejaoude mentioned wanting to collect tips and tricks from members of the community to generate short, 1 minute videos about the tip.

This could be done in a couple of ways:
- If you are comfortable with videos, editing, etc, you can record your own 1 minute video about a tip or trick.
- Alternatively, you can create an issue? discussion? something else? - and Eddie can produce a video with *your* tip or trick.

Share your thoughts below - what do you think? How should we implement this?

**Comment by eddiejaoude** [comment]
Thanks Nick for raising the discussion 👍 

I think to make it easier for everyone and keep the consistent style, it might be better for people to write a bullet point list of steps (I could come up with a template), then I can run through the steps to create the video and give the person a shoutout.

For example...

---

**Goal of the video**: Learn react basics
Steps:
1. install react cli with `npm ...`
2. run react cli command to create project and folder `react ...`
3. ...

**Comment by DevCassie** [comment]
I really like the idea of this.

I must say. I do agree with Eddie though. One format would be easier to follow.

**Reply by eddiejaoude** [reply]
Thanks, the first few will be the hardest, but will be fun 🤓

**Reply by DevCassie** [reply]
I think it's the hardest to come up with the format haha. But you've got this 💪

**Reply by eddiejaoude** [reply]
Thanks 🤓
